# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/reiserfsprogs/no-acl-hotfix.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- reiserfsprogs-3.6.27/reiserfscore/xattr.c.vanilla	2021-01-25 18:58:54.000629525 +0100
+++ reiserfsprogs-3.6.27/reiserfscore/xattr.c	2021-01-25 18:58:59.109629889 +0100
@@ -1,7 +1,7 @@
 #include <sys/types.h>
 #include <stdio.h>
 #include <errno.h>
-#include <sys/acl.h>
+//#include <sys/acl.h>
 #include "reiserfs_lib.h"
 
 static inline unsigned short from32to16(unsigned int x)
